conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreter.scala

```diff
@@ -46,6 +46,10 @@ case class MappingRuleInterpreter(rule: MappingConformanceRule, conformance: Con
     log.info(s"Processing mapping rule to conform ${rule.outputColumn}...")
     import spark.implicits._
 
+    //A fix for cases, where the join condition only uses columns previously created by a literal rule
+    //see https://github.com/AbsaOSS/enceladus/issues/892
+    spark.conf.set("spark.sql.crossJoin.enabled", "true")
```

Comment:
Well this works. But wouldn't it be better/nicer to do this on higher level, when the_Conformance_ rules are loaded?
* We would be setting the configuration only once.
* There would be no duplicate code (here and in `MappingRuleInterpreterGroupExplode.scala`
* The class `FeatureSwitches` could be leveraged to quickly and efficiently switch on/off the settting.

Reply:
Yes good point, by all means would be cleaner and nicer if configurable.

I would very much like to prioritize #895 though, since that would represent a proper fix for long term.